packages/office-ui-fabric-react/src/components/Checkbox/examples/Checkbox.Basic.Example.tsx

```diff
@@ -75,6 +79,12 @@ export class CheckboxBasicExample extends React.Component<{}, ICheckboxBasicExam
           boxSide='end'
           styles={ styles }
         />
+
+        <Checkbox
```

Comment:
Eh, i think we need to reorganize these examples anyway, so if you have time to do this, great, but not required.